src/array_api_extra/_delegators.py

```diff
@@ -0,0 +1,61 @@
+"""Delegators to existing implementations for Public API Functions."""
+
+from ._lib import _funcs
+from ._lib._utils._compat import (
+    array_namespace,
+    is_cupy_namespace,
+    is_jax_namespace,
+    is_numpy_namespace,
+    is_torch_namespace,
+)
+from ._lib._utils._typing import Array, ModuleType
```

Comment:
True for `ModuleType`, but since `Array` is "custom", I'd rather leave it like this I think. Then whenever we can start using things from `array-api-typing`, only that file needs to change.